src/Neo/Network/P2P/Payloads/Conflicts.cs

```diff
@@ -38,6 +41,23 @@ public override JObject ToJson()
 
         public override bool Verify(DataCache snapshot, Transaction tx)
         {
+            // Check already stored signers
+
+            var conflictingSigners = tx.Signers.Select(s => s.Account);
+
+            foreach (var attr in tx.GetAttributes<Conflicts>())
+            {
+                var conflictRecord = snapshot.TryGet(
+                    new KeyBuilder(NativeContract.Ledger.Id, LedgerContract.Prefix_Transaction).Add(attr.Hash))?
+                    .GetInteroperable<TransactionState>();
+                if (conflictRecord == null) continue;
+
+                if (conflictRecord.ConflictingSigners.Concat(conflictingSigners).Distinct().Count() > 100)
```

Comment:
Could we move `100` to some constant?  Maybe `MaxAllowedConflictingSigners` or something like that?

100 unique signers give us 200 byte of storage space, which seems good to me.

Reply:
FYI, to be more precise, 100 signers need 6+100*22 bytes

Reply:
Sure, and it's not much, we can safely allow this.